### PR TITLE
Init_seeder

### PR DESCRIPTION
**Pull request by pipinamngua**
Init seeder

### CODE REVIEW

**Comment by pipinamngua** [comment]
Init Seeder

**Comment by pipinamngua** [comment]
ready

File: database/seeds/RolesTableSeeder.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('roles')->insert([
+        	[
+        	    'name' => 'Admin',
+                'created_at' => new DateTime,
```
Comment (trinhductoan) [inline database/seeds/RolesTableSeeder.php]: indent

**Comment by trinhductoan** [comment]
commented

File: database/seeds/RolesTableSeeder.php (hunk 1 above)
Comment (pipinamngua) [reply database/seeds/RolesTableSeeder.php]: ok a

**Comment by pipinamngua** [comment]
Edit COnvention RolesTableSeeder.php

**Comment by pipinamngua** [comment]
ready

**Pull request merged**